apps/mull-api/src/app/channel/channel.resolver.ts

```diff
@@ -28,4 +35,21 @@ export class ChannelResolver {
   async getChannel(@Args('id', { type: /* istanbul ignore next */ () => Int }) id: number) {
     return this.channelService.getChannel(id);
   }
+
+  @Query(/* istanbul ignore next */ () => EventChannel)
+  @UseGuards(AuthGuard)
+  async getEventChannel(@Args('id', { type: /* istanbul ignore next */ () => Int }) id: number) {
+    return this.channelService.getEventChannel(id);
+  }
```

Comment:
How do we want to handle security for this? In #251 I implemented the logic already so it would be redundant for you to do it again here

Reply:
Done!